keyboards/tominabox1/le_chiffre/le_chiffre.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 tominabox1 (@tominabox1) and sickbabies
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define KEYLOG_LEN 11
+
+static const char PROGMEM code_to_name[53] = {
```

Comment:
You shouldnt define static variables here, as multiple compilation units will then get their own copy.